### DISCUSSION TITLE
Fill the remaining page with a block

### DISCUSSION THREAD

**Discussion by memchr** [typst/typst#4417] [Q&A] [answered]
How do I create a block that fills the rest of a page? I tried `height: 100%` but that overflows to the next page if the current page is not empty.
```typst
#lorem(100)
#block(fill: aqua, width: 100%, height: 100% )
```
![image](https://github.com/typst/typst/assets/118117622/e0d77169-2c9e-4ea3-b666-85622389ec87)

**Comment by laurmaedje** [comment, marked as answer]
`block(height: 1fr, ..)` would be what you want. It is not supported yet, but will be in the next release.

**Comment by rbq** [comment]
@laurmaedje Doesn't seem to work in 0.13.1. Is this still on the roadmap?

**Reply by TheLostLambda** [reply]
This seems to be working for me! What are you having issues with?

**Reply by rbq** [reply]
> This seems to be working for me! What are you having issues with?

Indeed, it seems to work. 😅 I might have confused `block()` and `box()`, or some other mistake.
